src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -932,6 +932,77 @@ class CreateTableStatementSegment(BaseSegment):
     )
 
 
+@postgres_dialect.segment()
+class CreateTableAsStatementSegment(BaseSegment):
+    """A `CREATE TABLE AS` statement.
+
+    As specified in https://www.postgresql.org/docs/13/sql-createtableas.html
+    """
+
+    type = "create_table_as_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        OneOf(
+            Sequence(
+                OneOf("GLOBAL", "LOCAL", optional=True),
+                Ref("TemporaryGrammar", optional=True),
```

Comment:
This optional shouldn't be true. You can't have global/local without temporary grammar.

Reply:
@WittierDinosaur This is replicating what exists in `CreateTableStatement`. Would the following work?: 
```
        OneOf(
            Ref("TemporaryGrammar"),
            Sequence(
                OneOf("GLOBAL", "LOCAL"),
                Ref("TemporaryGrammar"),
            ),
            "UNLOGGED",
            optional=True,
        ),
```

Reply:
All you need to do is remove the optional=True from Ref(TemporaryGrammar), leave it in for OneOf(GLOBAL, LOCAL)

Reply:
Done.